pkg/apis/extensions/v1beta1/conversion.go

```diff
@@ -157,3 +158,55 @@ func Convert_networking_IPBlock_To_v1beta1_IPBlock(in *networking.IPBlock, out *
 	copy(out.Except, in.Except)
 	return nil
 }
+
+func Convert_v1beta1_IngressBackend_To_networking_IngressBackend(in *extensionsv1beta1.IngressBackend, out *networking.IngressBackend, s conversion.Scope) error {
+	if err := autoConvert_v1beta1_IngressBackend_To_networking_IngressBackend(in, out, s); err != nil {
+		return err
+	}
+
+	if in.ServiceName != "" {
```

Comment:
need to set this if any of the service fields are populated:

```suggestion
	if in.ServiceName != "" || in.ServicePort.IntVal != 0 || in.ServicePort.StrVal != "" || in.ServicePort.Type == intstr.String {
```